dub.json

```diff
@@ -29,7 +29,7 @@
             "sourcePaths": ["dstep", "clang"],
             "importPaths": ["dstep", "clang"],
             "dflags-windows-x86": ["-m32mscoff"],
-            "lflags-posix": ["-lclang", "-rpath", ".", "-rpath", "/usr/lib/llvm-3.7/lib", "-L.", "-L/usr/lib64/llvm", "-L/usr/lib/llvm-3.7/lib"],
+            "lflags-posix": ["-lclang", "-rpath", ".", "-rpath", "/usr/lib/llvm-3.8/lib", "-L.", "-L/usr/lib64/llvm", "-L/usr/lib/llvm-3.8/lib"],
```

Comment:
Please don't modify this. I'll remove this eventually when I've finished support for static and dynamic linking of libclang.